README.md

```diff
@@ -260,6 +260,14 @@ You can populate below template according to your requirements and use it as you
 
 # AWS_STORAGE_CLASS="GLACIER"
 
+# Setting this variable will change the S3 default part size for the copy step.
+# This value is useful when you want to upload large files.
+# NB : While using Scaleway as S3 provider, be aware that the parts counter is set to 1.000.
+# While Minio uses a hard coded value to 10.000. As a workaround, try to set a higher value.
+# Defaults to "16" (MB) if unset (from minio), you can set this value according to your needs.
+
+# AWS_PART_SIZE=
```

Comment:
```suggestion
# AWS_PART_SIZE=1000
```

I think it'd be good to set an example value here so people know how the number is expected to be formatted. E.g. right now I'm not sure if I'm expected to pass `1.000` or `1000`.

Reply:
Okay, I'll take all in consideration and I will commit as soon as I can !